dll/win32/shimgvw/shimgvw.c

```diff
@@ -633,6 +651,7 @@ ImageView_DrawImage(HWND hwnd)
             GdipSetSmoothingMode(graphics, SmoothingModeHighQuality);
         }
 
+        SelectObject(hdc, GetStockObject(NULL_BRUSH));
```

Comment:
Should this be undone after? Not sure what the previous brush is but it seems like it could be leaked.

Reply:
```c
HGDIOBJ hbrOld;
...
hbrOld = SelectObject(hdc, GetStockObject(NULL_BRUSH));
...
SelectObject(hdc, hbrOld);
```